src/langchain_google_bigtable/document_loader.py

```diff
@@ -132,6 +146,20 @@ def lazy_load(
                         ].value,
                         mapping,
                     )
+            if (
+                self.metadata_as_json_column_family is not None
+                and self.metadata_as_json_column_name is not None
+                and self.metadata_as_json_column_family in row.cells
+                and self.metadata_as_json_column_name.encode()
+                in row.cells[self.metadata_as_json_column_family]
+            ):
+                metadata.update(
+                    json.loads(
+                        row.cells[self.metadata_as_json_column_family][
+                            self.metadata_as_json_column_name.encode()
+                        ][0].value.decode(self.metadata_as_json_encoding.value)
+                    )
+                )
```

Comment:
nit: this is like half the lines and twice as readable
```suggestion
                col_family, col_name = self.metadata_as_json_column_family, self.metadata_as_json_column_name
                r =  row.cells[col_family][col_name.encode()][0]
                j = json.loads(r.value.decode(self.metadata_as_json_encoding.value))
                metadata.update(json.loads(r.value.decode(self.metadata_as_json_encoding.value)))
```

Reply:
Done